pandas/util/_test_decorators.py

```diff
@@ -30,6 +30,7 @@ def test_foo():
 
 import numpy as np
 import pytest
+from _pytest.mark import MarkDecorator
```

Comment:
TODO: check if pytest types are public

Reply:
This is an issue we are working on: https://github.com/pytest-dev/pytest/issues/7469

I can't guarantee that we'll get it ready for pytest 6.0.0 itself, but we'll try.

In the meantime, it's better not to import from `_pytest` because that is prone to break without notice.

Reply:
OK will leave out these annotations for now. have added issue as comment.